src/features/editing/editingProvider.ts

```diff
@@ -34,4 +36,23 @@ export class EditingProvider {
 
     return builder.buildModelerView();
   }
+
+  private getDefaultBpmnDiagram(fileName:string):string
+  {
```

Comment:
```suggestion
  private getDefaultBpmnDiagram(fileName:string):string {
```

Reply:
I changed the style stuffs but I have not yet added any unit tests. I am not able to run the tests locally.